README.md

```diff
@@ -67,7 +67,7 @@ quickstart();
 
 ## Samples
 
-Samples are in the [`samples/`](https://github.com/googleapis/nodejs-local-auth/tree/master/samples) directory. Each sample's `README.md` has instructions for running its sample.
+Samples are in the [`samples/`](https://github.com/googleapis/nodejs-local-auth/tree/main/samples) directory. Each sample's `README.md` has instructions for running its sample.
```

Comment:
The docs job is probably failing here, because this URL doesn't exist yet 🙃

Reply:
sg